pkg/api/mlflow/dao/repositories/metric.go

```diff
@@ -2,20 +2,32 @@ package repositories
 
 import (
 	"context"
+	"database/sql"
 
 	"github.com/rotisserie/eris"
 	"gorm.io/gorm"
 	"gorm.io/gorm/clause"
 
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/api/request"
 	"github.com/G-Research/fasttrackml/pkg/api/mlflow/dao/models"
 	"github.com/G-Research/fasttrackml/pkg/database"
 )
 
+const (
+	MetricHistoriesDefaultLimitPerPage   = 10000000
+	MetricHistoryBulkDefaultLimitPerPage = 25000
```

Comment:
```suggestion
	MetricHistoriesDefaultLimit   = 10000000
	MetricHistoryBulkDefaultLimit = 25000
```

These are not pageable endpoints

Reply:
ok, done.